model/Image.php

```diff
@@ -233,10 +247,10 @@ public function SetRatioSize($width, $height) {
 		$heightRatio = $height / $this->height;
 		if( $widthRatio < $heightRatio ) {
 			// Target is higher aspect ratio than image, so check width
-			if($this->isWidth($width) && !Config::inst()->get('Image', 'force_resample')) return $this;
+			if($this->isWidth($width)) return $this;
```

Comment:
I have to disagree - calling a function which has an expected output format has now changed that output format. The promise for a certain file naming convention was implicit and enforced by the way the code has been written and run prior to this change. 

All of these functions are public API, we can't change an output format for them without it being a breaking change.